client/image_list_test.go

```diff
@@ -12,14 +12,15 @@ import (
 
 	"github.com/alibaba/pouch/apis/types"
 
+	"github.com/alibaba/pouch/apis/filters"
```

Comment:
put this line in the second part, thanks @ZYecho

Reply:
thanks, update.